book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+# Book class
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Let's add `rentals` to `attr_accessor` as user can read and write rentals too.

```suggestion
class Book
  attr_accessor :title, :author, :rentals
```